### PR TITLE
PMM-7 Fixed developer environment on m1 cpus

### PR DESCRIPTION
**Pull request by gen1us2k**
PMM-0

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

**Comment by gen1us2k** [comment]
@BupycHuk I found better solution than https://github.com/percona/pmm/pull/1104

File: docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: .
       args:
-        PMM_SERVER_IMAGE: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-container}
+        PMM_SERVER_IMAGE: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-latest}
       dockerfile: devcontainer.Dockerfile
```
Comment (BupycHuk) [inline docker-compose.yml:9]: Why don't just use `image` field instead of `build`?

Comment (gen1us2k) [reply docker-compose.yml:9]: Idk. Here's [changes related](https://github.com/percona/pmm/commit/0dfd5220b1b1a1c5262e20dacffddb4f1160faf5)

Comment (BupycHuk) [reply docker-compose.yml:9]: let's revert changes related to the docker-compose file and make your changes in the commented part.

**Review by BupycHuk — APPROVED** [review]

**Review by palash25 — APPROVED** [review]

**Pull request merged**